src/Scripting/AsTLTypes.hpp

```
@@ -0,0 +1,403 @@
+	inline std::string StackValueToRepString(const StackValue &stackVal) {
+		std::string rv{};
+		std::visit([&](const auto &val) {
+			rv = StackValueToRepString(typeid(decltype(val)));
+		}, stackVal);
+
+		return rv;
+	}
+
+
+	/* Gets the default value of a stack value type. */
+	inline StackValue GetDefaultStackValue(std::type_index type) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)